elastic/indexer/elastic-indexer.go

```diff
@@ -121,6 +127,23 @@ func (i *Indexer) runStep() error {
 	return nil
 }
 
+func (i *Indexer) checkAllResultMessagesAreValid(result *bulkResult) error {
+	for _, bulkResultOperation := range result.Items {
+		if bulkResultOperation.Index.ID != "" {
```

Comment:
If you define the type of `Index` as `*struct{...}` you can check if the pointer is nil ;) 
`json.Unmarshal` will still work correctly